lib/vanagon/platform/solaris_10.rb

```diff
@@ -42,14 +42,16 @@ def generate_package(project)
           # - The bin directory and all bin files are owned by root:bin instead of root:sys
           # - All files under lib are owned by root:bin instead of root:sys
           # - All .so files are owned by root:bin instead of root:sys
-          %((cd $(tempdir)/#{name_and_version}; pkgproto . | sort | awk ' \
```

Comment:
Oops, That is true. I didn't mean to remove the extra parens, though it didn't have any effect on the output, the build still worked and installing on solaris-10 still worked fine. However for posterity I will reintroduce the parens